docs/tutorials/tqdm_progress_bar.ipynb

```diff
@@ -278,6 +278,16 @@
         "**Below is the expected output when you run the cell above**\n",
```

Comment:
<p>you need to set verbose=0, otherwise there will be 2 progress bars.</p><p> </p><p>also if you have time, may you take a screenshot like the one above and link it in this colab? (because when we display the colab on tensorflow.org, the output of the cell won't be saved as display as desired, <a href="https://www.tensorflow.org/addons/tutorials/tqdm_progress_bar" target="_blank">https://www.tensorflow.org/addons/tutorials/tqdm_progress_bar</a>) this is optional, thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
@shun-lin, yes I tried to do that. But the markdown of the above image has path somewhere from rawgithubcontent. There is no asset folder in the repo. Where should I keep screenshot. Please help me. I don’t know

Reply:
https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d

It is kind of weird, but just take a screenshot, drag it into a comment in an issue (or anywhere like here should work) wait a few second until the url comes up. Then you can copy that URL into the colab).

Reply:
<img width="651" alt="evaluatepbar" src="https://user-images.githubusercontent.com/20843596/79412655-a57cef00-7fc3-11ea-9bea-ade8ee8dad58.PNG">